pom.xml

```diff
@@ -117,11 +118,13 @@
         <groupId>com.google.code.findbugs</groupId>
         <artifactId>jsr305</artifactId>
         <version>2.0.1</version>
+        <scope>compile</scope>
       </dependency>
       <dependency>
         <groupId>com.google.errorprone</groupId>
         <artifactId>error_prone_annotations</artifactId>
         <version>2.0.8</version>
+        <scope>compile</scope>
```

Comment:
I think compile is the default, and it means it's on the compilation and runtime classpath. You might be able to do something with optional, but I'd rather leave this as-is unless it's causing problems.